packages/@lightningjs/ui-components/src/components/FocusManager/FocusManager.mdx

```diff
@@ -119,35 +119,58 @@ class FancyFocus extends FocusManager {
 
 ### Properties
 
-| name          | type                                                    | readonly | description                                                                                                                  |
-| ------------- | ------------------------------------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| direction     | 'column'\|'row'                                         | false    | determines whether focus handling responds to left/right or up/down key                                                      |
-| selected      | <DocsLink id="lng.Component" />                         | true     | current item in focus                                                                                                        |
-| selectedIndex | number                                                  | false    | index of current item in focus                                                                                               |
-| loading       | boolean                                                 | true     | False if an item is in focus, true otherwise                                                                                 |
-| items         | <DocsLink id="lng.Component">lng.Component[]</DocsLink> | false    | alias for `children`                                                                                                         |
-| wrapSelected  | boolean                                                 | false    | enables wrapping behavior, so `selectNext` selects the first item if the current item is the last on the list and vice versa |
+| name          | type            | default   | description                                                    |
+| ------------- | --------------- | --------- | -------------------------------------------------------------- |
+| direction     | 'column'\|'row' | undefined | The navigation direction for focus ('left/right' or 'up/down') |
+| selectedIndex | number          | undefined | The index of the currently selected item.                      |
+| Items         | FocusItemsType  | undefined | Child element or elements of the FocusManager.                 |
```

Comment:
I believe for this property, the array of child elements is lower case `items`.

Rather than having `FocusItemsType` (without defining what that is), we can use the original `<DocsLink id="lng.Component">lng.Component[]</DocsLink>` as the type unless you'd like to define what `FocusItemsType` is in the docs.

Reply:
Ok, then I would prefer to use now `<DocsLink id="lng.Component">lng.Component[]</DocsLink> `